timelock/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "soroban-timelock-contract"
+version = "0.0.0"
+authors = ["Stellar Development Foundation <info@stellar.org>"]
+license = "Apache-2.0"
+edition = "2021"
+publish = false
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+doctest = false
+
+[features]
+default = ["export"]
+export = []
```

Comment:
❗ I don't think these features are needed for this example. We've been keeping them out of examples that don't need them. Can you remove them?
```suggestion
```

Reply:
Done.